### PR TITLE
Create PULL_REQUEST_TEMPLATE.md

### PR DESCRIPTION
**Pull request by prestosole**
.

### CODE REVIEW

**Comment by triallax** [comment]
Where did you get this template from? Can you credit them?

**Comment by prestosole** [comment]
That's a template, where to put the credit?

**Comment by triallax** [comment]
That will need thinking, but it's no excuse for not crediting the original authors.

I would put it in an HTML comment at the very beginning of the file:
```html
<!-- Put the attribution here -->
```
You might also credit them in the README, but I think you should do the HTML comment thing regardless of having the crediting in the README.

**Comment by prestosole** [comment]
Resolved!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<--! Template adapted from [auth0](https://github.com/auth0/open-source-template/blob/master/.github/PULL_REQUEST_TEMPLATE.md) -->
+
+By submitting a PR to this repository, you agree to the terms within our [Code of Conduct](https://github.com/lacerte/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/lacerte/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.
```
Comment (triallax) [inline .github/PULL_REQUEST_TEMPLATE.md]: The links are incorrect:
```suggestion
By submitting a PR to this repository, you agree to the terms within our [Code of Conduct](https://github.com/lacerte/clima/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/lacerte/clima/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.
```

**Review by triallax — CHANGES_REQUESTED** [review]

**Review by triallax — APPROVED** [review]

**Pull request merged**